tests/testTriggers.go

```diff
@@ -3182,6 +3184,77 @@ func getCloudSnapInterval(triggerType string) int {
 
 }
 
+// TriggerAppScaleUpAndDown performs app scale up and down according to chaos level
+func TriggerAppScaleUpAndDown(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: AppScaleUpAndDown,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	chaosLevel := ChaosMap[AppScaleUpAndDown]
+	var initialScaleFactor map[string]int32
+	context("scale apps up and down", func() {
+		for _, ctx := range *contexts {
+			for i := 0; i < chaosLevel; i++ {
+				customConfig, ok := Inst().CustomAppConfig[ctx.App.Key]
+				if ok {
+					Step("scaling up apps", func() {
+						applicationScaleUpMap, err := Inst().S.GetScaleFactorMap(ctx)
+						UpdateOutcome(event, err)
+						if err == nil {
+							for key, value := range applicationScaleUpMap {
+								initialScaleFactor[key] = value
+								key = strings.ReplaceAll(key, k8s.DeploymentSuffix, "")
+								key = strings.ReplaceAll(key, k8s.StatefulSetSuffix, "")
+								applicationScaleUpMap[key] = int32(customConfig.CustomPodConfig[key].Replicas)
```

Comment:
scale down to zero when this event is triggered. and scale up to original replica count at the event interval.

Reply:
delete all the pods in namespace and wait for them to spin up to exact replicas